pennylane/ops/qubit.py

```diff
@@ -223,6 +243,11 @@ def _matrix(cls, *params):
     def _eigvals(cls, *params):
         return np.array([1, 1j])
 
+    @staticmethod
+    def decomposition(wires):
+        decomp_ops = [RZ(np.pi / 2, wires=wires)]
```

Comment:
Note that by decomposing this in terms of the phase shift gate, we avoid the global phase:

```python
>>> np.round(qml.PhaseShift(np.pi / 2, wires=0).matrix, 3)
array([[1.+0.j, 0.+0.j],
       [0.+0.j, 0.+1.j]])
```

I'm wondering if it makes sense to decompose this in terms of `qml.PhaseGate` instead. Then, we can have the phase shift gate `qml.PhaseShift(phi)` get decomposed to `qml.RZ(phi)` up to a global phase if the device doesn't support phase shifts.